### ISSUE TITLE
Implement other DOMBackends to allow dropping jquery from projects (or un-pinning jquery version)

### ISSUE THREAD

**Issue by Tarang** [meteor/meteor#806]
If you create a blank project

>meteor create random
>cd random
>meteor remove jquery
>meteor

and then test jquery with

```javascript
console.log($().jquery)
```

it will output data indicating JQuery is still bundled

**Comment by amannn** [comment]
I would also really like to remove jquery. Modern browsers with better dom apis don't always need jquery.

**Comment by joetidee** [comment]
I too want to remove it. When i run 'meteor remove jquery' it tells me that it is not part of my project. I'm assuming that this is because it is required by Meteor, but the probem comes when I want to use a different version of jQuery in my project - how do we get around this issue?

**Comment by dcworldwide** [comment]
Seems the present version is jquery@1.11.3_2. For those of us that don't need IE6-8 support, it would be great if we could remove this version and replace it with the latest 2.x build. Is this achievable presently?

**Comment by glasser** [comment]
I believe the status quo remains: https://github.com/meteor/meteor/blob/devel/packages/blaze/dombackend.js is designed to allow other implementations of the functionality offered by JQuery but no other backends have been built.

**Comment by glasser** [comment]
(One idea would be, in lieu of actually implementing another backend, making `jquery` a non-release package and specifying its version only in `meteor-platform`.)

**Comment by ivankravchenko** [comment]
@glasser, could you mind providing an example how to pass own DOMBackend implementation into Blaze?

**Comment by glasser** [comment]
You can't right now, which is why this issue is open :)

**Comment by mquandalle** [comment]
(@glasser this issue is actually closed)

**Comment by ArjunRajJain** [comment]
any update on this, i wanted to see if i could use zepto.js instead

**Comment by zeroasterisk** [comment]
Agree - `jQuery` = 142KB & `zepto` = 54KB (uncompressed) 

It would at least be nice as an option to have a `meteor-platform-zepto` package, if functional.

**Comment by johanholmerin** [comment]
I experimented with replacing jQuery with pure JS in Blaze. It needs some more testing and an overlook by somebody more familiar with Blaze internals, but it seems to be working. However, since there is no API to change DOMBackend and jQuery is a hard dependency in Blaze and implied in meteor-platform, it's not very useful.

[Repo](https://github.com/johanholmerin/blaze-without-jquery)

**Comment by DmitryEfimenko** [comment]
I've removed jquery as a dependency from my project which uses angular for all DOM manipulation.
I have the following `<body>` tag in the `index.html`
```html
<body layout="column" id="{{'state-' + $root.$state.current.name}}">
```
Now getting error:
```
Uncaught TypeError: $ is not a function
```
... to the following code in the `template.index.js` that I assume gets generated by Meteor:
```javascript
Meteor.startup(function() { $('body').attr({"layout":"column","id":"{{'state-' + $root.$state.current.name}}"}); });
```

**Comment by stubailo** [comment]
@DmitryEfimenko this has been fixed: #5320 

closing this issue since it's not going to happen.
